tests/integration/integration_test.go

```diff
@@ -212,8 +212,11 @@ func getAllSyscallsInSet(set string) []string {
 	return syscallsInSet
 }
 
+// TODO This test is nice, but very hard to run in CI pipeline because it requires
+//      root privileges. I'll keep it for now so we can run it manually with sudo.
 func Test_Events(t *testing.T) {
-	t.Skip("This is an integration test")
+	t.Skip("This test requires root privileges")
+
```

Comment:
How can this be manually run?

Reply:
That's a very good question to the author of this test ;) I've added GoDoc with instructions on how to run in manually based on reverse engineering and taking into account new Make targets.